crypto/keys/keybase.go

```diff
@@ -413,9 +433,16 @@ func (kb dbKeybase) writeOfflineKey(pub tmcrypto.PubKey, name string) Info {
 
 func (kb dbKeybase) writeInfo(info Info, name string) {
 	// write the info by key
-	kb.db.SetSync(infoKey(name), writeInfo(info))
+	key := infoKey(name)
+	kb.db.SetSync(key, writeInfo(info))
+	// store a pointer to the infokey by address for fast lookup
+	kb.db.SetSync(addrKey(info.GetAddress()), key)
+}
+
+func addrKey(address types.AccAddress) []byte {
+	return []byte(fmt.Sprintf("%s.%s", address.String(), addressSuffix))
```

Comment:
what's the dot in the string for ?

Reply:
@fedekunze I think it's his way of creating a composite key. Would prefer to see something like `/` instead.

Reply:
why not the ol'reliable `:` instead ?

Reply:
Can do.

Reply:
I believe this is currently implemented as a ".". I'd prefer to not break the keybase without discussing it an in issue. (So we make sure that upgradability is maintained, and/or we roll more breaking keybase changes into it.) So lets leave it as a "." for now.

If we were to change the delimiter, we should use what IAVL uses for consistency imo. I believe that does some cool thing with key formats, which we could just import.